openapi.yaml

```diff
@@ -474,6 +474,167 @@ paths:
          responses:
             '200':
                description: Success
+      get:
+         summary: Get device
+         description: Get a device by id (serial number).
+         parameters:
+            - name: orgId
+              in: path
+              required: true
+              schema:
+                 type: string
+              description: The ID of the organization. Locate the orgId under Organization Settings > API Keys.
+            - name: deviceId
+              in: path
+              required: true
+              schema:
+                 type: string
+              description: The ID (serial number) of the device.
+         responses:
+            '200':
+               description: A list of devices
```

Comment:
Oops this returns a single device object rather than a "list of devices". Seems like a copy-paste holdover! Mind updating the response description @quan-managexr to say "A device object" or similar?